.coderabbit.yaml

```
@@ -0,0 +1,84 @@
+  path_filters:
+    - '!apps/classes/vendor/**'
+    - '!services/sumally-nx/**/gen/**'
+  path_instructions: [ ]
+  abort_on_close: true
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    ignore_title_keywords:
+      - 'release:'
+      - 'release('
+    labels: [ ]
```

yamllint

[error] too many spaces inside empty brackets

(brackets)